bundler/logger.go

```diff
@@ -9,9 +10,16 @@ import (
 	"github.com/orchestd/log/bzerolog"
 	"github.com/orchestd/sharedlib/consts"
 	"go.uber.org/fx"
+	"os"
 )
 
 func LoggerFxOption() fx.Option {
+	turnoffLogger := os.Getenv("turnoffLogger")
+	if turnoffLogger == "true" {
+		return fx.Options(
+			fx.Provide(emptyLogger.NewEmptyLogger),
```

Comment:
it looks like in this case you turn off line 58 as well
		trace.TraceInfoContextExtractorFxOption(),
is this ok ??

Reply:
yes, off everything about tracer